python/lsst/obs/comCam/ingest.py

```diff
@@ -84,7 +85,7 @@ def translate_wavelength(self, md):
         wl = int(round(raw_wl))
         if abs(raw_wl-wl)>=0.1:
             logger = lsstLog.Log.getLogger('obs.comCam.ingest')
-            logger.warn('Translated signigicatly non-integer wavelength %s', raw_wl)
+            logger.warn('Translated overly non-integer wavelength; %s is more than 0.1nm from an integer value', raw_wl)
```

Comment:
"Overly" is overly worse, I think. "significantly" (spelled correctly) is fine, particularly with the latter added comment.